modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/parser/DefaultCmmnActivityBehaviorFactory.java

```diff
@@ -82,7 +82,7 @@ public MilestoneActivityBehavior createMilestoneActivityBehavior(PlanItem planIt
         } else if (StringUtils.isNotEmpty(milestone.getName())) {
             name = milestone.getName();
         }
-        return new MilestoneActivityBehavior(expressionManager.createExpression(name), milestone.getMilestoneVariable());
+        return new MilestoneActivityBehavior(expressionManager.createExpression(name), milestone.getMilestoneVariable(),milestone.getBusinessStatusUpdate());
```

Comment:
Here is another missing space:
```suggestion
        return new MilestoneActivityBehavior(expressionManager.createExpression(name), milestone.getMilestoneVariable(), milestone.getBusinessStatusUpdate());
```